client/layout/guided-tours/config-elements/link-quit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import contextTypes from '../context-types';
+
+export default class LinkQuit extends Component {
+	static propTypes = {
+		children: PropTypes.node,
+		primary: PropTypes.bool,
+		subtle: PropTypes.bool,
+		href: PropTypes.string,
+		target: PropTypes.string,
+	};
+
+	static contextTypes = contextTypes;
+
+	constructor( props, context ) {
+		super( props, context );
+	}
+
+	onClick = ( event ) => {
+		this.props.onClick && this.props.onClick( event );
+		const { quit, tour, tourVersion, step, isLastStep } = this.context;
+		quit( { tour, tourVersion, step, isLastStep } );
+	}
+
+	render() {
+		const { children, primary, subtle, href, target } = this.props;
+		const classes = subtle ? 'guided-tours__subtle-button' : '';
+		return (
+			<Button
+				className={ classes }
+				onClick={ this.onClick }
+				primary={ primary }
+				href={ href }
+				target={ target }>
+				{ children || translate( 'Quit' ) }
+			</Button>
+		);
+	}
+}
```

Comment:
There wasn't a good way to include a button that was also a link and also quit the tour. Either option we want to close the tour after the action.

If you're aware of another way to handle this, I'd appreciate it if you could point me to it.